test/evergreen.yml

```diff
@@ -3370,6 +3370,21 @@ tasks:
         vars:
           perf-test-name: modify-force-update-large-record-btree.wtperf
 
+  - name: perf-test-mongodb-oplog
+    tags: ["btree-perf"]
```

Comment:
This feels like a good candidate for a new `oplog-perf` tag if we make the changes from my other comment.

Reply:
Agreed